lib/capistrano/asg/tasks/rolling.rake

```diff
@@ -125,7 +125,7 @@ namespace :rolling do
       end
     else
       logger.error 'No instances have been launched. Are you using a configuration with rolling deployments?'
-      exit 1
+      raise 'Auto Scaling Group update failed'
```

Comment:
This rake task (launch_instances) does not do any changes on the ASG, so I'd prefer to keep it as is (exit 1). Or do you have a specific reason to change it?

Reply:
I updated to follow capistrano conventions, happy to undo if you'd prefer?

Reply:
I guess that makes sense.. but then maybe replace the logger.error with the raise ?